project_recalculate/models/project_project.py

```diff
@@ -56,12 +56,12 @@ def project_recalculate(self):
             if not project.calculation_type:
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have calculation type."))
-            if (project.calculation_type == 'date_begin'
-                    and not project.date_start):
+            if (project.calculation_type == 'date_begin' and not
+                    project.date_start):
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have date start."))
-            if (project.calculation_type == 'date_end'
-                    and not project.date):
+            if (project.calculation_type == 'date_end' and not
+                    project.date):
```

Comment:
I guess this is related to https://github.com/OCA/project/pull/150, merged in the meanwhile.